migrate.py

```diff
@@ -205,7 +205,9 @@ def visit_Assign(self, node):
     deps = []
     for fragment in doc_finder.fragments:
         try:
-            fragment_collection = get_plugin_collection(fragment, 'doc_fragments', spec)
+            # some doc_fragments use subsections (e.g. vmware.vcenter_documentation)
+            fragment_name = fragment.split('.')[0]
```

Comment:
this might not be correct when it is already referencing a collection, but I don't think that will be the case for anything in ansible/ansible